components/Pill.tsx

```diff
@@ -5,26 +5,56 @@ type PillProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
 	size?: 'small' | 'medium' | 'large';
 	color?: 'yellow' | 'gray' | 'red';
 	outline?: boolean;
+	fullWidth?: boolean;
+	isRounded?: boolean;
+	blackFont?: boolean;
```

Comment:
black font is a bit of weird name for a prop as we have light and dark theme

Reply:
Thanks. `black` means font weight here. It's not intuitive.  Do you have any other suggestion?
![image](https://user-images.githubusercontent.com/4819006/232996607-d61de6ab-8fa2-4bfa-9c70-370e78018943.png)

Reply:
I will just name it as `weight` same as the `Body` component.